charts/sonarqube/values.yaml

```diff
@@ -72,17 +72,21 @@ networkPolicy:
   # expects https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#networkpolicyspec-v1-networking-k8s-io
   # additionalNetworkPolicys:
 
+# will be used as default for ingress path and probes path, will be injected in .Values.env as SONAR_WEB_CONTEXT
+# if .Values.env.SONAR_WEB_CONTEXT is set, this value will be ignored
+sonarWebContext: ""
+
 # also install the nginx ingress helm chart
 nginx:
   enabled: false
 
 ingress:
   enabled: false
   # Used to create an Ingress record.
-  hosts:
-    - name: sonarqube.your-org.com
+  # hosts:
```

Comment:
We should uncomment these lines, to meet the default values we have in the readme (It could be that was me commenting them out)